src/quadrature/quadrature.hpp

```diff
@@ -12,7 +12,7 @@
 #include "ddc_helper.hpp"
 
 /**
- * @brief A class providing an operator for integrating functions defined on a discrete index range.
+ * @brief A class providing an operator for integrating functions defined on a discrete domain.
```

Comment:
Do we still want to talk about "discrete domain"s? I thought we agreed that this terminology was confusing.

```suggestion
 * @brief A class providing an operator for integrating functions defined on known grid points.
```
?

Reply:
Yes, it is probably better. But "discrete index range" was a bit weird x)

Reply:
Definitely! But if we're fixing docstrings then we might as well completely fix them

Reply:
Unresolving as changes do not match suggestion and are ungrammatical